src/com/twu/biblioteca/BibliotecaApp.java

```diff
@@ -1,8 +1,12 @@
 package com.twu.biblioteca;
 
+import com.twu.biblioteca.WelcomeMessage;
+
 public class BibliotecaApp {
 
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+
+        WelcomeMessage.welcome();
```

Comment:
Nice!  (As usual) I have a few things to keep in mind about this:
- Because the `main` method is `static`, it can be tempting to make everything it calls be `static` as well, which "works", but is edging closer to using global data.  Keep looking for a tipping point where we should be creating objects (i.e., `new WelcomeMessage()`) rather than calling static methods on it.
- I would also regularly ask the question of whether `WelcomeMessage` is doing enough to warrant a separate class?  If we decided to delete WelcomeMessage (but keep the requirement), where would you put the code that is currently within it?

Reply:
Thanks, would the below be correct?
`WelcomeMessage welcomeMessage = new WelcomeMessage();`
`welcomeMessage.welcome();`

I think I can put the code into the MainMenu class I have created.

Reply:
Yep - that's correct.  Most of your Java career will be creating objects from classes and then calling methods on those objects.  The early stories don't really force you into this approach, but it's good to get comfortable with it.

Reply:
Awesome, thanks! It's good to start getting into the habit now.